docs/source/notebooks/release_announcement.ipynb

```diff
@@ -0,0 +1,2774 @@
+{
```

Comment:
<p>We could also say that we support other types of plots like cost curves which are in-build inside the package. Still, the user can extract the posterior and PyMC model to per for custom analysis.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Added two sentences at the end of the MMM section</p>